homeassistant/helpers/intent.py

```diff
@@ -681,6 +691,13 @@ def async_match_targets(  # noqa: C901
                     final_candidates.extend(group_candidates)
                     continue
 
+            # Try to disambiguate by exposure
+            exposed_candidates = [c for c in group_candidates if c.is_exposed]
+            if len(exposed_candidates) < 2:
+                # Only one candidate was exposed
+                final_candidates.extend(exposed_candidates)
+                continue
+
```

Comment:
That is then not needed.

Reply:
Thanks for the help! Much appreciated :slightly_smiling_face: